Fix NASA Scrub in CI

NASA Scrub scans are failing in CI. This issue is to fix it. [This PR](https://github.com/NASA-AMMOS/aerie/pull/534) is a temporary fix so we can keep moving with other PRs.

```
Run python3 -m pip install nasa-scrub
Defaulting to user installation because normal site-packages is not writeable
Collecting nasa-scrub
  Downloading nasa_scrub-2.6.1-py2.py3-none-any.whl (55 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.8/55.8 kB 1.8 MB/s eta 0:00:00
Installing collected packages: nasa-scrub
Successfully installed nasa-scrub-2.6.1
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/runner/work/scrub/scrub/scrub/tools/parsers/csv_parser.py", line 66, in <module>
    parse_warnings(pathlib.Path(sys.argv[1]).resolve())
  File "/home/runner/work/scrub/scrub/scrub/tools/parsers/csv_parser.py", line 57, in parse_warnings
    result.get('file').relative_to(source_dir),
  File "/usr/lib/python3.10/pathlib.py", line 818, in relative_to
    raise ValueError("{!r} is not in the subpath of {!r}"
ValueError: '/home/runner/work/scrub/scrub/scrub/targets/collaborator/do_collaborator.py' is not in the subpath of '/home/runner/work/scrub/results' OR one path is relative and the other is absolute.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/codeql.yml
+++ b/.github/workflows/codeql.yml
@@ -14,6 +14,7 @@ name: "CodeQL"
 on:
   push:
     branches: [ develop ]
+    tags: [ v* ]
   pull_request:
     # The branches below must be a subset of the branches above
     branches: [ develop ]
@@ -75,14 +76,24 @@ jobs:
       uses: github/codeql-action/analyze@v2
 
     - name: NASA Scrub
-      continue-on-error: true
       run: |
-        pip3 install nasa-scrub
-        python3 -m scrub.tools.parsers.translate_results /home/runner/work/aerie/results/*.sarif /home/runner/work/aerie/results/codeql.scrub ${{ github.workspace }} scrub
-        python3 -m scrub.tools.parsers.csv_parser /home/runner/work/aerie/results
+        python3 -m pip install nasa-scrub
+
+        results_dir=`realpath ${{ github.workspace }}/../results`
+        sarif_files=`find $results_dir -name '*.sarif'`
+
+        for sarif_file in $sarif_files
+        do
+          output_file="$results_dir/$(basename $sarif_file .sarif).scrub"
+          python3 -m scrub.tools.parsers.translate_results $sarif_file $output_file ${{ github.workspace }} scrub
+        done
+
+        python3 -m scrub.tools.parsers.csv_parser $results_dir
+        
+        echo "RESULTS_DIR=$results_dir" >> $GITHUB_ENV
 
     - name: Upload CodeQL Artifacts
       uses: actions/upload-artifact@v3
       with:
-        name: codeql-artifacts
-        path: /home/runner/work/aerie/results/*
+        name: code-scan-results
+        path: ${{ env.RESULTS_DIR }}

